### DISCUSSION TITLE
Lua file integration is not working in windows command prompt

### DISCUSSION THREAD

**Discussion by bysayem** [JanDeDobbeleer/oh-my-posh#1275] [Troubleshoot]
Lua integration is not working, I have installed latest clink and added oh-my-posh.lua to the clink's script  directory, It shows me error like thish, how I can fix it?

![Screenshot (2)](https://user-images.githubusercontent.com/58431777/142778842-2af5048d-e94a-4a3f-96d0-aff21d52af2c.png)

**Comment by JanDeDobbeleer** [comment]
@sayem-sadat do you have a space in your User folder? As that's what I'm seeing there. Can you share the content of the Lua file?

**Reply by bysayem** [reply]
Sir, I have deleted the entire windows of my laptop, for performing e clean window 11 installation, In my pc, I will try clink and oh-my-posh. If I face any issue I will inform you
